addons/sale/models/sale.py

```diff
@@ -325,6 +339,15 @@ def onchange_partner_id(self):
         values['team_id'] = partner_user.team_id.id if partner_user and partner_user.team_id else self.team_id
         self.update(values)
 
+    def _inverse_expected_date(self):
+        """ When writing on expected_date, if the value should be modify manually (`expected_date_mode` = 'manual' only),
+            then we put the value in `expected_date_manual`. Otherwise, `expected_date_manual` should be False since the
+            expected date is automatically compute by other mecanisms.
+        """
+        for order in self:
+            order.expected_date_mode = 'manual'
+            order.expected_date_manual = order.expected_date
```

Comment:
in an inverse method you're writing on expected_date_manual which is a stored field.. doesn't look good